deps/v8/include/v8.h

```diff
@@ -5574,6 +6169,11 @@ class V8_EXPORT EmbedderHeapTracer {
    */
   virtual void AbortTracing() {}
 
+  /**
+   * Returns the number of wrappers that are still to be traced by the embedder.
+   */
+  virtual size_t NumberOfWrappersToTrace() { return 0; }
```

Comment:
I suppose it is technically possible for an add-on to call `Isolate::SetEmbedderHeapTracer()` but I can't see that being useful: an add-on won't know how to trace objects it didn't create itself.

(Neither does core know about objects created by add-ons.  This API doesn't seem all that usable inside the node ecosystem, it seems to assume the embedder is a single monolithic entity.)